models/Game.js

```diff
@@ -19,10 +28,75 @@
 
 module.exports = class Game {
   constructor(userWhoStartsGame) {
+    this.gameId = null;
+    this.socketId = null;
     this.players = [userWhoStartsGame];
+    this.seatsOpen = settings.maxPlayers - 1;
     this.dealer = null;
-    this.solution = null;
+    this.solutionForDisplay = null;
+    this.solutionForMatcing = null;
     this.category = null;
     this.clue = [];
+    this.active = false;
+  }
+
+  setOpenSeats() {
```

Comment:
the "set" word implies that I'm feeding it a value, like `this.setOpenSeats(5)`, perhaps rename to `updateSeatCount`